components/ui/autocomplete/build.gradle

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+}
+
+dependencies {
+    compile "com.android.support:appcompat-v7:27.1.0"
+
+    compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    compileOnly "com.android.support:support-compat:27.1.0"
+
+    testImplementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    testImplementation 'junit:junit:4.12'
+    testImplementation "org.robolectric:robolectric:3.7.1"
+    testImplementation 'org.mockito:mockito-core:2.12.0'
+}
+
+archivesBaseName = "autocomplete"
+
+apply from: '../../../publish.gradle'
+ext.configurePublish(
+        'org.mozilla.components',
+        'ui-autocomplete',
```

Comment:
So far I have used the browser/ui/support prefixes for structuring the repository and gradle modules. But I haven't used them in the artifact id (e.g. support-ktx -> org.mozilla.components:ktx).
Do you think we should add them? If yes, then maybe to the group id instead? Or keeping the current naming scheme for now and then let's revisit later when we have more components (and avoiding updating all the configurations and remote repositories).

Reply:
True, if we wanted to keep them, it would have to be a group id. I will remove for now to keep it simple. We can always add additional structure later if we need it.